### ISSUE TITLE
[Android] If the minimumDate is set on 'datetime' mode, then when you open the datepicker, it shows the minimum date, not the currentDate

### ISSUE THREAD

**Issue by catalinbotean** [henninghall/react-native-date-picker#633]
If I set the minimumDate on Android, when I open the datepicker it will show me the dates starting from the minimum date (instead of showing me the current date like it happens on iOS). So for example, I set the minimumDate to be '2020-01-01' and the currentDate is '2022-04-03'. When I open the datepicker on Android I see this (1st January 2020 is the minimum date, not the selected one).
<img width="412" alt="android" src="https://user-images.githubusercontent.com/62332851/230671681-6249872e-4fd1-4987-8d99-13d50be1bc0e.png">
On the other hand, on iOS I see this:
<img width="340" alt="ios" src="https://user-images.githubusercontent.com/62332851/230671938-a4b5e11e-959f-4004-94dc-fce8f56b17a1.png">

The expected behaviour would be to see when I open the datepicker on 'datetime' mode the selected date, even if the minimumDate is set.

```
<DatePicker
	mode="datetime"
	date={selectedDate}
	onDateChange={handleDateChange}
	androidVariant="nativeAndroid"
	minuteInterval={minuteInterval}
	maximumDate={maximumDate}
	minimumDate={minimumDate}
/>
```

 - OS: Android
 - React Native version: 0.70.7
 - react-native-date-picker version: 4.2.9

**Comment by henninghall** [comment]
Thanks for reporting this! a fix has been released in `4.2.12`.

**Comment by angelinaromashkoo** [comment]
Unfortunately I have the same issue, the version is 4.3.3
